tasking/pulp/tasking/services/scheduler.py

```diff
@@ -10,14 +10,13 @@
 from django.db import IntegrityError
 
 from pulp.app.models.task import TaskLock, Worker
-from pulp.tasking import worker_watcher
-from pulp.tasking.celery_instance import celery as app
+from pulp.tasking.celery_app import celery as app
 from pulp.tasking.constants import TASKING_CONSTANTS as constants
-from pulp.tasking import delete_worker
+from pulp.tasking.services import worker_watcher
 
 
 # The import below is not used in this module, but it needs to be kept here. This module is the
-# first and only Pulp module to be imported by celerybeat, and by importing pulp.server.logs, it
+# first and only Pulp module to be imported by celerybeat, and by importing pulp.app.logs, it
```

Comment:
I suspect this comment and the import on L#21 is probably out of date and can likely be removed along with the [`pulp.app.logs`](https://github.com/pulp/pulp/blob/3.0-dev/app/pulp/app/logs.py).